ltac/tacinterp.ml

```diff
@@ -1422,7 +1428,9 @@ and tactic_of_value ist vle =
         extra = TacStore.set ist.extra f_trace []; } in
       let tac = name_if_glob appl (eval_tactic ist t) in
       Profile_ltac.do_profile "tactic_of_value" trace (catch_error_tac trace tac)
-  | (VFun _|VRec _) -> Tacticals.New.tclZEROMSG (str "A fully applied tactic is expected.")
+  | VFun (_, _, _,vars,_) ->
+    Tacticals.New.tclZEROMSG (str "A fully applied tactic is expected: missing argument(s) " ++ arg_names vars)
```

Comment:
Is it worth casing the "argument"/"arguments" distinction on the length of the list?